### ISSUE TITLE
Have a problem about StandardScaler

### ISSUE THREAD

**Issue by gayo66** [AIStream-Peelout/flow-forecast#482]
Thanks for your code which help me a lot.
And I want to ask a question about StandardScaler:
When creating a data loader, shouldn't the mean and standard deviation of the training set be used for validation and test set standardization? 
I see in the code that the validation set uses the mean and standard deviation of the validation set, and the test set uses the mean and standard deviation of the test set.
Hope for your reply. Thanks!

**Comment by isaacmg** [comment]
Yeah it probably should in general as good practice most of the datasets historically we have used though have had similar distributions so it hasn't really affected things. But we will probably fix in an upcoming PR.
